CATALOG.md

```diff
@@ -0,0 +1,42 @@
+# Build Commands Library
+
+Below you'll find a list of build commands contributed from the nativefier community.
+
+If you'd like to add to this catalog, please keep the following in mind:
+
+Library commands should be the strict necessary to make an app work. So, for example,
```

Comment:
Compacting these two things into a single point, and adding a point asking to not add trivial sites.

```suggestion
- Only add sites that require something special! No need to document here that `simplesite.com` works with a simple `nativefier simplesite.com` 🙂
- If you'd like to add to this catalog, please add commands with the *strict necessary* to make an app work. So, for example,
```